src/emqtt_bench.erl

```diff
@@ -819,39 +826,41 @@ publish(Client, Opts) ->
     ok = ensure_publish_begin_time(),
     Flags   = [{qos, proplists:get_value(qos, Opts)},
                {retain, proplists:get_value(retain, Opts)}],
-   Payload0 = proplists:get_value(payload, Opts),
```

Comment:
This diff is because I also fixed indentation for this block. Let me know if you would rather have this in a separate commit or PR.